src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();

```

Comment:
The 'fruits' map is declared as public which can be accessed and modified from anywhere in the application. Consider making it private and providing public methods to interact with it.

Reply:
ok, done